legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/postprocessor/defaultPostProcessor/dbSpecificProcessor.pure

```diff
@@ -30,11 +30,11 @@ import meta::relational::functions::sqlQueryToString::*;
 function meta::relational::mapping::dbSpecificProcessor(query:SQLQuery[1], runtime:Runtime[1], store:Database[0..1], exeCtx:ExecutionContext[1], extensions:Extension[*]): PostProcessorResult[1]
 {
    let connectionStore = $runtime.connectionStores->filter(c|$c.connection->instanceOf(DatabaseConnection));
-   let conn = $connectionStore.connection->cast(@DatabaseConnection);
+   let conn = $connectionStore.connection->distinct()->cast(@DatabaseConnection);
    let postProcessorQuery = if($conn->isNotEmpty(),
                                 | let type = $conn.type;
                                   assert(type->size()==1, 'found not enough or too many DB types');
-                                  if ($type->toOne()->createDbConfig([]).dbExtension.aliasLimit->isNotEmpty(),
+                                  if ($type->distinct()->toOne()->createDbConfig([]).dbExtension.aliasLimit->isNotEmpty(),
```

Comment:
this distinct is odd, as the line above will fail if size is not 1.  Either distinct is not required, or should be done above.